samples/Microsoft.AspNetCore.Proxy.Samples/Microsoft.AspNetCore.Proxy.Samples.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
```

Comment:
Done. btw. Should we remove:
<!-- TODO remove when https://github.com/dotnet/sdk/issues/396 is resolved -->
    <RuntimeIdentifier Condition=" '$(TargetFramework)' != 'netcoreapp1.1' ">win7-x64</RuntimeIdentifier>
as it seems to be already resolved? (Just asking not sure about this)